src/main/java/wtf/meier/tariff/interpreter/helper/serializer/CurrencySerializer.kt

```diff
@@ -0,0 +1,18 @@
+package wtf.meier.tariff.interpreter.helper.serializer
```

Comment:
Can we put this into wtf.meier.tariff.interpreter.serialization ? Or why do we have the helper middle package here?

Reply:
My intention was, if we have more than the serializer as a helper, we can put it there. But then we can also change that later.